apps/frontend/src/app/3_organisms/TransactionHistoryFrame/TransactionHistoryFrame.tsx

```diff
@@ -528,7 +538,16 @@ export const TransactionHistoryFrame: FC = () => {
 
   const exportData = useCallback(() => {
     if (!troves) {
-      alert(t(translations.transactionHistory.actions.noDataToExport));
+      addNotification(
+        {
+          type: NotificationType.success,
```

Comment:
```suggestion
          type: NotificationType.warning,
```
I'd probably use `warning` (or `error` if you want) because it looks weird with `success` styling.